frontend/public/components/storage/attach-storage.tsx

```diff
@@ -0,0 +1,437 @@
+/* eslint-disable no-undef */
+import * as _ from "lodash-es";
+import * as React from "react";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import {
+  k8sGet,
+  k8sUpdate,
+  K8sResourceKind,
+  referenceFor,
+} from "../../module/k8s";
+import * as k8sModels from "../../models";
+import {
+  ButtonBar,
+  history,
+  resourceObjPath,
+  ListDropdown,
+  ResourceIcon
+} from "../utils";
+import { formatNamespacedRouteForResource } from "../../ui/ui-actions";
+import {
+  getMountPaths,
+  createVolumeMount,
+  createVolume,
+  getKindFromPlural
+} from "../storage/storage";
+
+const Checkbox = ({ value, checked, onChange }) => (
+  <div>
+    <label className="checkbox-label">
+      <input
+        type="checkbox"
+        onChange={({ target: { checked: newChecked } }) =>
+          onChange(value, newChecked)
+        }
+        checked={!!checked}
+      />
+      &nbsp;&nbsp;
+      {value}
+    </label>
+  </div>
```

Comment:
Let's use the Bootstrap checkbox styles. This should probably be in a separate component for reuse.

```js
<div className="form-group">
  <div className="checkbox">
    <label className="control-label">
      <input type="checkbox" ...>
      {value}
    </label>
  </div>
</div>
```